myScript.js

```diff
@@ -39,4 +39,165 @@ function createList() {
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  }, {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+];
+
+function createProjectPopUp() {
+  const x = parseInt(this.id[4], 10) - 1;
+  const blurryBackground = document.createElement('div');
+  blurryBackground.setAttribute('class', 'blurry-background');
+
+  const mainwindow = document.createElement('div');
+  mainwindow.setAttribute('class', 'main-window ');
+  blurryBackground.appendChild(mainwindow);
+  const div1 = document.createElement('div');
+  // div1-p2w4
+  div1.setAttribute('class', 'div1-p2w4');
+  mainwindow.appendChild(div1);
+  const title = document.createElement('h1');
+  // title-p2w4
+  title.setAttribute('class', 'title-p2w4');
+  const titleText = document.createTextNode(projects[x].title);
+  title.appendChild(titleText);
+  div1.appendChild(title);
+  const closetapimg = document.createElement('img');
+  // close-p2w4
+  closetapimg.setAttribute('class', 'close-p2w4');
+  closetapimg.setAttribute('src', 'img/x-lg.svg');
+  closetapimg.setAttribute('id', 'btn-d2'); // modify
+  div1.appendChild(closetapimg);
+  closetapimg.onclick = () => blurryBackground.remove();
+  const div2 = document.createElement('div');
+  div2.setAttribute('class', 'div2-p2w4');
+  mainwindow.appendChild(div2);
+  const tech = projects[x].technologies;
+  for (let i = 0; i < 3; i += 1) {
+    const a = document.createElement('a');
+    // tech-p2w4
+    a.setAttribute('class', 'tech-p2w4');
+    const buttontext = document.createTextNode(tech[i]);
+    a.appendChild(buttontext);
+    div2.appendChild(a);
+  }
+  const div3 = document.createElement('div');
+  div3.setAttribute('class', 'div3-p2w4');
+  mainwindow.appendChild(div3);
```

Comment:
- [ ] Your popup is showing the same data whenever I open something. Please make it load data the correct data :blush:

Reply:
@HAMMAS-SALEEM  
but the Figma provides only one version of the data how can I get different images and different names for different projects please explain

Reply:
@HAMMAS-SALEEM  
I also asked some of the Micronauts that have already gotten approval they said that they used the same version of data for all the projects

Reply:
Hey, @yasinabdmahmood I saw that which you mentioned. There is no problem with your project regarding this change. You can go ahead without taking this change into consideration. :blush: